crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -525,37 +525,52 @@ fn depth_props_ui(
                 );
             ui.end_row();
 
-            ui.label("Backproject scale");
-            let mut scale = *entity_props.backproject_scale.get();
-            let speed = (scale * 0.05).at_least(0.01);
-            if ui
+            {
+                ui.label("Backproject meter");
+                let mut meter = *entity_props.backproject_depth_meter.get();
+                let speed = (meter * 0.05).at_least(0.01);
+                let response = ui
                 .add(
-                    egui::DragValue::new(&mut scale)
+                    egui::DragValue::new(&mut meter)
                         .clamp_range(0.0..=1.0e8)
                         .speed(speed),
                 )
-                .on_hover_text("Scales the backprojected point cloud")
-                .changed()
-            {
-                entity_props.backproject_scale = EditableAutoValue::UserEdited(scale);
+                .on_hover_text("How many depth values correspond to one work-space unit. For instance, 1000 means millimeters.\
```

Comment:
```suggestion
                .on_hover_text("How many unit steps in the depth texture correspond to one work-space unit. For instance, 1000 means millimeters.\
```

"how many depth values" reads to me like how many samples in the depth textures and that's very strange.
Why 1000 means milimeters is not clear to me, but again don't want to open that can of worms right now :)